### ISSUE TITLE
No keyboard or mouse when connected via xorg

### ISSUE THREAD

**Issue by synaptis** [neutrinolabs/xorgxrdp#164]
Client: Windows 10 build 19541.1000
Server: Fedora 30 Workstation

Package versions:
xorgxrdp-0.2.13-1.fc30.x86_64
xrdp-selinux-0.9.13-1.fc30.x86_64
xrdp-0.9.13-1.fc30.x86_64

Issue:
When connecting to the Fedora 30 machine via mstsc and  xorg the connection and login are successful, the desktop displays and updates (MS Teams opens by default and I can see it update) but the mouse and keyboard inputs are not registered. I have left a connection in this state for 5 minutes and without the issue resolving. Eventually I have to close the mstsc client.

I have tried this with both my laptop keyboard and trackpad and separate USB connected keyboard and mouse. Also tried lowering the quality/bandwidth in the mstsc settings.

Connections with the same configuration over xvnc do not show this issue. Xorg is preferred as it recognises my monitors as separate screens rather than a single spanned screen.

/var/log/xrdp-sesman.log:
```
[20200401-08:49:50] [INFO ] A connection received from 127.0.0.1 port 49610
[20200401-08:49:50] [INFO ] ++ created session (access granted): username det\EZ002260, ip 10.1.38.123:65257 - socket: 12
[20200401-08:49:50] [INFO ] starting Xorg session...
[20200401-08:49:50] [DEBUG] Closed socket 13 (AF_INET 0.0.0.0:5910)
[20200401-08:49:50] [DEBUG] Closed socket 13 (AF_INET 0.0.0.0:6010)
[20200401-08:49:50] [DEBUG] Closed socket 13 (AF_INET 0.0.0.0:6210)
[20200401-08:49:50] [DEBUG] Closed socket 10 (AF_INET 127.0.0.1:3350)
[20200401-08:49:50] [INFO ] calling auth_start_session from pid 2445
[20200401-08:49:50] [DEBUG] Closed socket 9 (AF_INET 127.0.0.1:3350)
[20200401-08:49:50] [DEBUG] Closed socket 10 (AF_INET 127.0.0.1:3350)
[20200401-08:49:50] [INFO ] setpriv --no-new-privs Xorg :10 -auth .Xauthority -config xrdp/xorg.conf -noreset -nolisten tcp -logfile .xorgxrdp.%s.log  
[20200401-08:49:50] [CORE ] waiting for window manager (pid 2448) to exit
```
/var/log/xrdp.log:
```
[20200401-08:49:33] [DEBUG] Closed socket 12 (AF_INET 10.25.64.107:3389)
[20200401-08:49:33] [DEBUG] Closed socket 11 (AF_INET 0.0.0.0:3389)
[20200401-08:49:33] [DEBUG] TLSv1.3 enabled
[20200401-08:49:33] [DEBUG] TLSv1.2 enabled
[20200401-08:49:33] [DEBUG] Security layer: requested 11, selected 1
[20200401-08:49:33] [DEBUG] Closed socket 12 (AF_INET 10.25.64.107:3389)
[20200401-08:49:34] [INFO ] Socket 12: AF_INET connection received from 10.1.38.123 port 65257
[20200401-08:49:34] [DEBUG] Closed socket 12 (AF_INET 10.25.64.107:3389)
[20200401-08:49:34] [DEBUG] Closed socket 11 (AF_INET 0.0.0.0:3389)
[20200401-08:49:34] [DEBUG] TLSv1.3 enabled
[20200401-08:49:34] [DEBUG] TLSv1.2 enabled
[20200401-08:49:34] [DEBUG] Security layer: requested 11, selected 1
[20200401-08:49:34] [INFO ] connected client computer name: NOMAD
[20200401-08:49:34] [INFO ] adding channel item name rdpdr chan_id 1004 flags 0x80800000
[20200401-08:49:34] [INFO ] adding channel item name rdpsnd chan_id 1005 flags 0xc0000000
[20200401-08:49:34] [INFO ] adding channel item name cliprdr chan_id 1006 flags 0xc0a00000
[20200401-08:49:34] [INFO ] adding channel item name drdynvc chan_id 1007 flags 0xc0800000
[20200401-08:49:34] [INFO ] TLS connection established from 10.1.38.123 port 65257: TLSv1.2 with cipher ECDHE-RSA-AES256-GCM-SHA384
[20200401-08:49:34] [DEBUG] xrdp_0000098b_wm_login_mode_event_00000001
[20200401-08:49:34] [INFO ] Loading keymap file /etc/xrdp/km-00000409.ini
[20200401-08:49:34] [WARN ] local keymap file for 0x00000409 found and doesn't match built in keymap, using local keymap file
[20200401-08:49:50] [DEBUG] xrdp_wm_log_msg: connecting to sesman ip 127.0.0.1 port 3350
[20200401-08:49:50] [INFO ] xrdp_wm_log_msg: sesman connect ok
[20200401-08:49:50] [DEBUG] xrdp_wm_log_msg: sending login info to session manager, please wait...
[20200401-08:49:50] [DEBUG] return value from xrdp_mm_connect 0
[20200401-08:49:50] [INFO ] xrdp_wm_log_msg: login successful for display 10
[20200401-08:49:50] [DEBUG] xrdp_wm_log_msg: started connecting
[20200401-08:49:50] [INFO ] lib_mod_log_peer: xrdp_pid=2443 connected to X11rdp_pid=2449 X11rdp_uid=1072685679 X11rdp_gid=1072600513 client_ip=10.1.38.123 client_port=65257
[20200401-08:49:50] [DEBUG] xrdp_wm_log_msg: connected ok
[20200401-08:49:50] [DEBUG] xrdp_mm_connect_chansrv: chansrv connect successful
[20200401-08:49:50] [DEBUG] Closed socket 18 (AF_INET 127.0.0.1:49610)
[20200401-08:55:47] [DEBUG] Closed socket 12 (AF_INET 10.25.64.107:3389)
[20200401-08:55:47] [DEBUG] xrdp_mm_module_cleanup
[20200401-08:55:47] [DEBUG] Closed socket 19 (AF_UNIX)
[20200401-08:55:47] [DEBUG] Closed socket 20 (AF_UNIX)
```
J

/etc/xrdp/startwm.sh:
```
#!/bin/bash
if [ -r /etc/default/locale ]; then
  . /etc/default/locale
  export LANG LANGUAGE
fi
gnome-session
```
/etc/xorg/xorg.ini:
```
/etc/xrdp/xrdp.ini                                                                                            8931  08:35:27  
[Globals]
; xrdp.ini file version number
ini_version=1

; fork a new process for each incoming connection
fork=true
; tcp port to listen
port=3389
; 'port' above should be connected to with vsock instead of tcp
use_vsock=false
; regulate if the listening socket use socket option tcp_nodelay
; no buffering will be performed in the TCP stack
tcp_nodelay=true
; regulate if the listening socket use socket option keepalive
; if the network connection disappear without close messages the connection will be closed
tcp_keepalive=true
#tcp_send_buffer_bytes=32768
#tcp_recv_buffer_bytes=32768

; security layer can be 'tls', 'rdp' or 'negotiate'
; for client compatible layer
security_layer=negotiate
; minimum security level allowed for client
; can be 'none', 'low', 'medium', 'high', 'fips'
crypt_level=high
; X.509 certificate and private key
; openssl req -x509 -newkey rsa:2048 -nodes -keyout key.pem -out cert.pem -days 365
certificate=/etc/xrdp/cert.pem
key_file=/etc/xrdp/key.pem
; set SSL protocols
; can be comma separated list of 'SSLv3', 'TLSv1', 'TLSv1.1', 'TLSv1.2', 'TLSv1.3'
ssl_protocols=TLSv1.2, TLSv1.3
; set TLS cipher suites
#tls_ciphers=HIGH

; Section name to use for automatic login if the client sends username
; and password. If empty, the domain name sent by the client is used.
; If empty and no domain name is given, the first suitable section in
; this file will be used.
autorun=

allow_channels=true
allow_multimon=true
bitmap_cache=true
bitmap_compression=true
bulk_compression=true
#hidelogwindow=true
max_bpp=32
new_cursors=true
; fastpath - can be 'input', 'output', 'both', 'none'
use_fastpath=both
; when true, userid/password *must* be passed on cmd line
#require_credentials=true
; You can set the PAM error text in a gateway setup (MAX 256 chars)
#pamerrortxt=change your password according to policy at http://url

;
; colors used by windows in RGB format
;
blue=009cb5
grey=dedede
#black=000000
#dark_grey=808080
#blue=08246b
#dark_blue=08246b
#white=ffffff
#red=ff0000
#green=00ff00
#background=626c72

;
; configure login screen
;

; Login Screen Window Title
#ls_title=My Login Title

; top level window background color in RGB format
ls_top_window_bg_color=009cb5

; width and height of login screen
ls_width=350
ls_height=430

; login screen background color in RGB format
ls_bg_color=dedede

; optional background image filename (bmp format).
#ls_background_image=

; logo
; full path to bmp-file or file in shared folder
ls_logo_filename=
ls_logo_x_pos=55
ls_logo_y_pos=50

; for positioning labels such as username, password etc
ls_label_x_pos=30
ls_label_width=65

; for positioning text and combo boxes next to above labels
ls_input_x_pos=110
ls_input_width=210

; y pos for first label and combo box
ls_input_y_pos=220

; OK button
ls_btn_ok_x_pos=142
ls_btn_ok_y_pos=370
ls_btn_ok_width=85
ls_btn_ok_height=30

; Cancel button
ls_btn_cancel_x_pos=237
ls_btn_cancel_y_pos=370
ls_btn_cancel_width=85
ls_btn_cancel_height=30

[Logging]
LogFile=xrdp.log
LogLevel=DEBUG
EnableSyslog=true
SyslogLevel=DEBUG
; LogLevel and SysLogLevel could by any of: core, error, warning, info or debug

[Channels]
; Channel names not listed here will be blocked by XRDP.
; You can block any channel by setting its value to false.
; IMPORTANT! All channels are not supported in all use
; cases even if you set all values to true.
; You can override these settings on each session type
; These settings are only used if allow_channels=true
rdpdr=true
rdpsnd=true
drdynvc=true
cliprdr=true
rail=true
xrdpvr=true
tcutils=true

; for debugging xrdp, in section xrdp1, change port=-1 to this:
#port=/tmp/.xrdp/xrdp_display_10

; for debugging xrdp, add following line to section xrdp1
#chansrvport=/tmp/.xrdp/xrdp_chansrv_socket_7210


;
; Session types
;

; Some session types such as Xorg, X11rdp and Xvnc start a display server.
; Startup command-line parameters for the display server are configured
; in sesman.ini. See and configure also sesman.ini.
[Xvnc]
name=Xvnc
lib=libvnc.so
username=ask
password=ask
ip=127.0.0.1
port=-1
#xserverbpp=24
#delay_ms=2000

[Xorg]
name=Xorg
lib=libxup.so
username=ask
password=ask
ip=127.0.0.1
port=-1
code=20

#[vnc-any]
#name=vnc-any
#lib=libvnc.so
#ip=ask
#port=ask5900
#username=na
#password=ask
#pamusername=asksame
#pampassword=asksame
#pamsessionmng=127.0.0.1
#delay_ms=2000

#[neutrinordp-any]
#name=neutrinordp-any
#lib=libxrdpneutrinordp.so
#ip=ask
#port=ask3389
#username=ask
#password=ask

; You can override the common channel settings for each session type
#channel.rdpdr=true
#channel.rdpsnd=true
#channel.drdynvc=true
#channel.cliprdr=true
#channel.rail=true
#channel.xrdpvr=true
```
jounnalctl :
```
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: (5000)(140684849538880)[INFO ] ++ created session (access granted): username det\my_user, ip 10.1.38.123:65309 - socket: 12
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: (5000)(140684849538880)[INFO ] starting Xorg session...
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: (5000)(140684849538880)[DEBUG] Closed socket 13 (AF_INET 0.0.0.0:5910)
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: (5000)(140684849538880)[DEBUG] Closed socket 13 (AF_INET 0.0.0.0:6010)
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: (5000)(140684849538880)[DEBUG] Closed socket 13 (AF_INET 0.0.0.0:6210)
Apr 01 08:58:03 fedora30.domain.local xrdp[5371]: (5371)(139667214824960)[INFO ] xrdp_wm_log_msg: login successful for display 10
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: (5000)(140684849538880)[DEBUG] Closed socket 10 (AF_INET 127.0.0.1:3350)
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5375]: (5375)(140684849538880)[INFO ] calling auth_start_session from pid 5375
Apr 01 08:58:03 fedora30.domain.local xrdp[5371]: (5371)(139667214824960)[DEBUG] xrdp_wm_log_msg: started connecting
Apr 01 08:58:03 fedora30.domain.local audit[5375]: CRED_ACQ pid=5375 uid=0 auid=4294967295 ses=4294967295 subj=system_u:system_r:unconfined_service_t:s0 msg='op=PAM:setcred grantors=pam_localuser,pam_sss acct="det\my_user" exe="/usr/sbin/xrdp-sesman" hostname=? addr=? terminal=:10 res=success'
Apr 01 08:58:03 fedora30.domain.local systemd-logind[1168]: New session c3 of user my_user.
Apr 01 08:58:03 fedora30.domain.local systemd[1]: Started Session c3 of user my_user.
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5375]: pam_unix(xrdp-sesman:session): session opened for user det\my_user by (uid=0)
Apr 01 08:58:03 fedora30.domain.local audit[5375]: USER_START pid=5375 uid=0 auid=4294967295 ses=4294967295 subj=system_u:system_r:unconfined_service_t:s0 msg='op=PAM:session_open grantors=pam_keyinit,pam_limits,pam_systemd,pam_unix,pam_sss acct="det\my_user" exe="/usr/sbin/xrdp-sesman" hostname=? addr=? terminal=:10 res=success'
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5375]: (5375)(140684849538880)[DEBUG] Closed socket 9 (AF_INET 127.0.0.1:3350)
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5375]: (5375)(140684849538880)[DEBUG] Closed socket 10 (AF_INET 127.0.0.1:3350)
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5380]: (5380)(140684849538880)[INFO ] setpriv --no-new-privs Xorg :10 -auth .Xauthority -config xrdp/xorg.conf -noreset -nolisten tcp -logfile .xorgxrdp.%s.log
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: X.Org X Server 1.20.6
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: X Protocol Version 11, Revision 0
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: Build Operating System:  5.3.11-300.fc31.x86_64
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: Current Operating System: Linux fedora30.domain.local 5.3.15-200.fc30.x86_64 #1 SMP Thu Dec 5 15:18:00 UTC 2019 x86_64
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: Kernel command line: BOOT_IMAGE=(hd0,gpt2)/vmlinuz-5.3.15-200.fc30.x86_64 root=/dev/mapper/fedora-root ro resume=/dev/mapper/fedora-swap rd.lvm.lv=fedora/root rd.lvm.lv=fedora/swap rhgb quiet rd.driver.blacklist=nouveau modprobe.blacklist=nouveau nvidia-drm.modeset=1
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: Build Date: 25 November 2019  04:20:51PM
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: Build ID: xorg-x11-server 1.20.6-1.fc30
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: Current version of pixman: 0.38.0
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]:         Before reporting problems, check http://wiki.x.org
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]:         to make sure that you have the latest version.
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: Markers: (--) probed, (**) from config file, (==) default setting,
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]:         (++) from command line, (!!) notice, (II) informational,
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]:         (WW) warning, (EE) error, (NI) not implemented, (??) unknown.
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: (++) Log file: ".xorgxrdp.10.log", Time: Wed Apr  1 08:58:03 2020
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: (++) Using config file: "/etc/X11/xrdp/xorg.conf"
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: (==) Using config directory: "/etc/X11/xorg.conf.d"
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: (==) Using system config directory "/usr/share/X11/xorg.conf.d"
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: xorgxrdpSetup:
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: xrdpdevSetup:
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: rdpIdentify:
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: rdpDriverFunc: op 10
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: rdpProbe:
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: rdpPreInit:
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: rdpScreenInit: virtualX 800 virtualY 600 rgbBits 8 depth 24
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: rdpScreenInit: pfbMemory bytes 1920000
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: rdpScreenInit: pfbMemory 0x7f7cbbc18010
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: rdpSimdInit: assigning yuv functions
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: rdpSimdInit: cpuid ax 1 cx 0 return ax 0x000906e9 bx 0x04100800 cx 0x7ffafbff dx 0xbfebfbff
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: rdpSimdInit: sse2 amd64 yuv functions assigned
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: rdpClientConInit: disconnect idle session after [0] sec
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: rdpClientConInit: kill disconnected [1] timeout [60] sec
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: rdpXvInit: depth 24
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: rdpScreenInit: out
Apr 01 08:58:03 fedora30.domain.local xrdp[5371]: (5371)(139667214824960)[INFO ] lib_mod_log_peer: xrdp_pid=5371 connected to X11rdp_pid=5380 X11rdp_uid=1234567890 X11rdp_gid=1072600513 client_ip=10.1.38.123 client_port=65309
Apr 01 08:58:03 fedora30.domain.local xrdp[5371]: (5371)(139667214824960)[DEBUG] xrdp_wm_log_msg: connected ok
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: rdpCreateScreenResources:
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: rdpDeferredRandR:
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: rdpResizeSession: width 1024 height 768
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]:   calling RRScreenSizeSet
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: rdpRRScreenSetSize: width 1024 height 768 mmWidth 271 mmHeight 203
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: rdpRRGetInfo:
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]:   screen resized to 1024x768
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]:   RRScreenSizeSet ok 1
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: rdpResizeSession: width 2560 height 1440
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]:   calling RRScreenSizeSet
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: rdpRRScreenSetSize: width 2560 height 1440 mmWidth 677 mmHeight 381
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: rdpRRGetInfo:
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]:   screen resized to 2560x1440
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]:   RRScreenSizeSet ok 1
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: rdpClientConGotConnection:
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: rdpClientConGotConnection: g_sck_accept ok new_sck 9
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: rdpClientConGetConnection: idle_disconnect_timeout set to non-positive value, idle timer turned off
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: rdpAddClientConToDev: adding first clientCon 0x55cb53a60b70
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: rdpClientConProcessMsgVersion: version 0 0 0 1
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: rdpClientConProcessScreenSizeMsg: set width 2560 height 1440 bpp 24
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: rdpClientConProcessScreenSizeMsg: shmemid 26 shmemptr 0x7f7c8f1f0000
Apr 01 08:58:03 fedora30.domain.local xrdp-sesman[5000]: rdpClientConProcessMsgClientInput: invalidate x 0 y 0 cx 2560 cy 1440
Apr 01 08:58:04 fedora30.domain.local xrdp-sesman[5375]: (5375)(140684849538880)[CORE ] waiting for window manager (pid 5379) to exit
Apr 01 08:58:04 fedora30.domain.local xrdp[5371]: (5371)(139667214824960)[DEBUG] xrdp_mm_connect_chansrv: chansrv connect successful
Apr 01 08:58:04 fedora30.domain.local xrdp-sesman[5000]: rdpClientConProcessMsgClientInfo:
Apr 01 08:58:04 fedora30.domain.local xrdp-sesman[5000]:   got client info bytes 5752
Apr 01 08:58:04 fedora30.domain.local xrdp-sesman[5000]:   jpeg support 0
Apr 01 08:58:04 fedora30.domain.local xrdp-sesman[5000]:   offscreen support 1
Apr 01 08:58:04 fedora30.domain.local xrdp-sesman[5000]:   offscreen size 7864320
Apr 01 08:58:04 fedora30.domain.local xrdp-sesman[5000]:   offscreen entries 100
Apr 01 08:58:04 fedora30.domain.local xrdp-sesman[5000]:   client can not do offscreen to offscreen blits
Apr 01 08:58:04 fedora30.domain.local xrdp-sesman[5000]:   client can do new(color) cursor
Apr 01 08:58:04 fedora30.domain.local xrdp-sesman[5000]:   client can not do multimon
Apr 01 08:58:04 fedora30.domain.local xrdp-sesman[5000]: rdpRRSetRdpOutputs: numCrtcs 0 numOutputs 0 monitorCount 0
Apr 01 08:58:04 fedora30.domain.local xrdp-sesman[5000]: rdpRRSetRdpOutputs: add output 0 left 0 top 0 width 2560 height 1440
Apr 01 08:58:04 fedora30.domain.local xrdp[5371]: (5371)(139667214824960)[DEBUG] Closed socket 18 (AF_INET 127.0.0.1:51064)
Apr 01 08:58:04 fedora30.domain.local systemd[2048]: Starting Accessibility services bus...
Apr 01 08:58:04 fedora30.domain.local systemd[2048]: Started Accessibility services bus.
Apr 01 08:58:04 fedora30.domain.local at-spi-bus-launcher[5550]: Policy to allow eavesdropping in /usr/share/defaults/at-spi2/accessibility.conf +15: Eavesdropping is deprecated and ignored
Apr 01 08:58:04 fedora30.domain.local at-spi-bus-launcher[5550]: Policy to allow eavesdropping in /usr/share/defaults/at-spi2/accessibility.conf +17: Eavesdropping is deprecated and ignored
Apr 01 08:58:04 fedora30.domain.local dbus-broker-lau[5555]: Ready
Apr 01 08:58:04 fedora30.domain.local systemd[2048]: Created slice dbus\x2d:1.9\x2dorg.a11y.atspi.Registry.slice.
Apr 01 08:58:04 fedora30.domain.local systemd[2048]: Started dbus-:1.9-org.a11y.atspi.Registry@0.service.
Apr 01 08:58:04 fedora30.domain.local at-spi2-registryd[5558]: SpiRegistry daemon is running with well-known name - org.a11y.atspi.Registry
Apr 01 08:58:04 fedora30.domain.local gnome-session[5379]: gnome-session-check-accelerated: GL Helper exited with code 512
Apr 01 08:58:04 fedora30.domain.local gnome-session[5379]: libEGL warning: DRI2: failed to authenticate
Apr 01 08:58:04 fedora30.domain.local gnome-session[5379]: gnome-session-check-accelerated: GLES Helper exited with code 512
Apr 01 08:58:04 fedora30.domain.local gnome-keyring-daemon[5584]: couldn't access control socket: /run/user/1234567890/keyring/control: No such file or directory
Apr 01 08:58:04 fedora30.domain.local gnome-keyring-daemon[5585]: couldn't access control socket: /run/user/1234567890/keyring/control: No such file or directory
Apr 01 08:58:04 fedora30.domain.local gnome-keyring-d[5584]: couldn't access control socket: /run/user/1234567890/keyring/control: No such file or directory
Apr 01 08:58:04 fedora30.domain.local gnome-keyring-d[5585]: couldn't access control socket: /run/user/1234567890/keyring/control: No such file or directory
Apr 01 08:58:04 fedora30.domain.local gnome-keyring-daemon[5590]: couldn't access control socket: /run/user/1234567890/keyring/control: No such file or directory
Apr 01 08:58:04 fedora30.domain.local gnome-keyring-d[5590]: couldn't access control socket: /run/user/1234567890/keyring/control: No such file or directory
Apr 01 08:58:04 fedora30.domain.local gnome-keyring-ssh.desktop[5580]: SSH_AUTH_SOCK=/run/user/1234567890/keyring/ssh
Apr 01 08:58:04 fedora30.domain.local gnome-keyring-pkcs11.desktop[5582]: SSH_AUTH_SOCK=/run/user/1234567890/keyring/ssh
Apr 01 08:58:04 fedora30.domain.local gnome-keyring-secrets.desktop[5583]: SSH_AUTH_SOCK=/run/user/1234567890/keyring/ssh
Apr 01 08:58:04 fedora30.domain.local xrdp-sesman[5000]: rdpRRGetInfo:
Apr 01 08:58:04 fedora30.domain.local systemd[2048]: Starting Virtual filesystem service...
Apr 01 08:58:04 fedora30.domain.local systemd[2048]: Started Virtual filesystem service.
Apr 01 08:58:04 fedora30.domain.local org.gnome.Shell.desktop[5594]: Xlib:  extension "DPMS" missing on display ":10.0".
Apr 01 08:58:04 fedora30.domain.local xrdp-sesman[5000]: rdpRRScreenSetSize: width 2560 height 1440 mmWidth 677 mmHeight 381
Apr 01 08:58:04 fedora30.domain.local xrdp-sesman[5000]: rdpRRGetInfo:
Apr 01 08:58:04 fedora30.domain.local xrdp-sesman[5000]:   screen resized to 2560x1440
Apr 01 08:58:04 fedora30.domain.local org.gnome.Shell.desktop[5594]: Xlib:  extension "DPMS" missing on display ":10.0".
Apr 01 08:58:04 fedora30.domain.local xrdp-sesman[5000]: rdpRROutputSetProperty:
Apr 01 08:58:04 fedora30.domain.local xrdp-sesman[5000]: rdpRROutputGetProperty:
Apr 01 08:58:05 fedora30.domain.local systemd[2048]: Created slice dbus\x2d:1.1\x2dorg.freedesktop.portal.IBus.slice.
Apr 01 08:58:05 fedora30.domain.local systemd[2048]: Started dbus-:1.1-org.freedesktop.portal.IBus@0.service.
Apr 01 08:58:05 fedora30.domain.local systemd[2048]: Starting sandboxed app permission store...
Apr 01 08:58:05 fedora30.domain.local systemd[2048]: Started sandboxed app permission store.
Apr 01 08:58:05 fedora30.domain.local systemd[2048]: Created slice dbus\x2d:1.1\x2dorg.gnome.Shell.CalendarServer.slice.
Apr 01 08:58:05 fedora30.domain.local systemd[2048]: Started dbus-:1.1-org.gnome.Shell.CalendarServer@0.service.
Apr 01 08:58:05 fedora30.domain.local systemd[2048]: Starting Evolution source registry...
Apr 01 08:58:05 fedora30.domain.local systemd[2048]: Created slice dbus\x2d:1.1\x2dorg.gnome.OnlineAccounts.slice.
Apr 01 08:58:05 fedora30.domain.local systemd[2048]: Started dbus-:1.1-org.gnome.OnlineAccounts@0.service.
Apr 01 08:58:05 fedora30.domain.local systemd[2048]: Created slice dbus\x2d:1.1\x2dca.desrt.dconf.slice.
Apr 01 08:58:05 fedora30.domain.local systemd[2048]: Started dbus-:1.1-ca.desrt.dconf@0.service.
Apr 01 08:58:05 fedora30.domain.local systemd[2048]: Starting Virtual filesystem metadata service...
Apr 01 08:58:05 fedora30.domain.local systemd[2048]: Started Virtual filesystem metadata service.
Apr 01 08:58:05 fedora30.domain.local systemd[2048]: Started Evolution source registry.
Apr 01 08:58:05 fedora30.domain.local gnome-shell[5594]: g_dir_open_with_errno: assertion 'path != NULL' failed
Apr 01 08:58:05 fedora30.domain.local gnome-shell[5594]: g_filename_to_utf8: assertion 'opsysstring != NULL' failed
Apr 01 08:58:05 fedora30.domain.local gnome-shell[5594]: g_dir_open_with_errno: assertion 'path != NULL' failed
Apr 01 08:58:05 fedora30.domain.local gnome-shell[5594]: g_filename_to_utf8: assertion 'opsysstring != NULL' failed
Apr 01 08:58:05 fedora30.domain.local goa-daemon[5665]: goa-daemon version 3.32.1 starting
Apr 01 08:58:05 fedora30.domain.local systemd[1]: Starting Location Lookup Service...
Apr 01 08:58:05 fedora30.domain.local systemd[2048]: Created slice dbus\x2d:1.1\x2dorg.gnome.Identity.slice.
Apr 01 08:58:05 fedora30.domain.local systemd[2048]: Started dbus-:1.1-org.gnome.Identity@0.service.
Apr 01 08:58:05 fedora30.domain.local goa-identity-se[5687]: GoaKerberosIdentityManager: Using polling for change notification for credential cache type 'KCM'
Apr 01 08:58:05 fedora30.domain.local systemd[1]: Started Location Lookup Service.
Apr 01 08:58:05 fedora30.domain.local audit[1]: SERVICE_START pid=1 uid=0 auid=4294967295 ses=4294967295 subj=system_u:system_r:init_t:s0 msg='unit=geoclue comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
Apr 01 08:58:05 fedora30.domain.local systemd[2048]: Created slice dbus\x2d:1.1\x2dorg.gnome.keyring.SystemPrompter.slice.
Apr 01 08:58:05 fedora30.domain.local systemd[2048]: Started dbus-:1.1-org.gnome.keyring.SystemPrompter@0.service.
Apr 01 08:58:05 fedora30.domain.local polkitd[1139]: Registered Authentication Agent for unix-session:c3 (system bus name :1.844 [/usr/bin/gnome-shell], object path /org/freedesktop/PolicyKit1/AuthenticationAgent, locale en_AU.UTF-8)
Apr 01 08:58:05 fedora30.domain.local gnome-shell[5594]: Telepathy is not available, chat integration will be disabled.
Apr 01 08:58:05 fedora30.domain.local systemd[2048]: Starting Virtual filesystem service - disk device monitor...
Apr 01 08:58:05 fedora30.domain.local gcr-prompter[5707]: bus acquired: org.gnome.keyring.SystemPrompter
Apr 01 08:58:05 fedora30.domain.local gcr-prompter[5707]: Gcr: registering prompter
Apr 01 08:58:05 fedora30.domain.local gcr-prompter[5707]: bus acquired: org.gnome.keyring.PrivatePrompter
Apr 01 08:58:05 fedora30.domain.local gcr-prompter[5707]: acquired name: org.gnome.keyring.SystemPrompter
Apr 01 08:58:05 fedora30.domain.local gcr-prompter[5707]: acquired name: org.gnome.keyring.PrivatePrompter
Apr 01 08:58:05 fedora30.domain.local gnome-keyring-daemon[5584]: couldn't create system prompt: Operation was cancelled
Apr 01 08:58:05 fedora30.domain.local gnome-keyring-d[5584]: couldn't create system prompt: Operation was cancelled
Apr 01 08:58:05 fedora30.domain.local goa-daemon[5665]: secret_password_lookup_sync() returned NULL
Apr 01 08:58:05 fedora30.domain.local goa-daemon[5665]: /org/gnome/OnlineAccounts/Accounts/account_1565915761_1: Setting AttentionNeeded to TRUE because EnsureCredentials() failed with: No credentials found in the keyring (goa-error-quark, 4)
Apr 01 08:58:05 fedora30.domain.local gcr-prompter[5707]: Gcr: received BeginPrompting call from callback /org/gnome/keyring/Prompt/p3@:1.20
Apr 01 08:58:05 fedora30.domain.local gcr-prompter[5707]: Gcr: preparing a prompt for callback /org/gnome/keyring/Prompt/p3@:1.20
Apr 01 08:58:05 fedora30.domain.local gcr-prompter[5707]: Gcr: creating new GcrPromptDialog prompt
Apr 01 08:58:05 fedora30.domain.local gcr-prompter[5707]: GLib-GIO: _g_io_module_get_default: Found default implementation gvfs (GDaemonVfs) for ‘gio-vfs’
Apr 01 08:58:05 fedora30.domain.local systemd[2048]: Started Virtual filesystem service - disk device monitor.
Apr 01 08:58:05 fedora30.domain.local systemd[2048]: Starting Virtual filesystem service - Media Transfer Protocol monitor...
Apr 01 08:58:05 fedora30.domain.local systemd[2048]: Started Virtual filesystem service - Media Transfer Protocol monitor.
Apr 01 08:58:05 fedora30.domain.local systemd[2048]: Starting Virtual filesystem service - digital camera monitor...
Apr 01 08:58:05 fedora30.domain.local systemd[2048]: Started Virtual filesystem service - digital camera monitor.
Apr 01 08:58:05 fedora30.domain.local systemd[2048]: Starting Virtual filesystem service - Apple File Conduit monitor...
Apr 01 08:58:05 fedora30.domain.local gvfs-afc-volume-monitor[5730]: Volume monitor alive
Apr 01 08:58:05 fedora30.domain.local systemd[2048]: Started Virtual filesystem service - Apple File Conduit monitor.
Apr 01 08:58:05 fedora30.domain.local systemd[2048]: Starting Virtual filesystem service - GNOME Online Accounts monitor...
Apr 01 08:58:05 fedora30.domain.local gcr-prompter[5707]: Gcr: automatically selecting secret exchange protocol
Apr 01 08:58:05 fedora30.domain.local gcr-prompter[5707]: Gcr: generating public key
Apr 01 08:58:05 fedora30.domain.local gcr-prompter[5707]: Gcr: beginning the secret exchange: [sx-aes-1]\npublic=/xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx\n
Apr 01 08:58:05 fedora30.domain.local gcr-prompter[5707]: Gcr: calling the PromptReady method on /org/gnome/keyring/Prompt/p3@:1.20
Apr 01 08:58:05 fedora30.domain.local gcr-prompter[5707]: Gcr: returned from the PromptReady method on /org/gnome/keyring/Prompt/p3@:1.20
Apr 01 08:58:05 fedora30.domain.local systemd[2048]: Started Virtual filesystem service - GNOME Online Accounts monitor.
Apr 01 08:58:05 fedora30.domain.local gcr-prompter[5707]: Gcr: received PerformPrompt call from callback /org/gnome/keyring/Prompt/p3@:1.20
Apr 01 08:58:05 fedora30.domain.local gcr-prompter[5707]: Gcr: receiving secret exchange: [sx-aes-1]\npublic=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx\n
Apr 01 08:58:05 fedora30.domain.local gcr-prompter[5707]: Gcr: deriving shared transport key
Apr 01 08:58:05 fedora30.domain.local gcr-prompter[5707]: Gcr: deriving transport key
Apr 01 08:58:05 fedora30.domain.local gcr-prompter[5707]: Gcr: starting password prompt for callback /org/gnome/keyring/Prompt/p3@:1.20
Apr 01 08:58:05 fedora30.domain.local systemd[1]: Starting PackageKit Daemon...
Apr 01 08:58:05 fedora30.domain.local PackageKit[5739]: daemon start
Apr 01 08:58:05 fedora30.domain.local audit[1]: SERVICE_START pid=1 uid=0 auid=4294967295 ses=4294967295 subj=system_u:system_r:init_t:s0 msg='unit=packagekit comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
Apr 01 08:58:05 fedora30.domain.local systemd[1]: Started PackageKit Daemon.
Apr 01 08:58:05 fedora30.domain.local dbus-broker[1125]: A security policy denied :1.121 to send method call /org/freedesktop/PackageKit:org.freedesktop.DBus.Properties.GetAll to :1.874.
Apr 01 08:58:05 fedora30.domain.local gsd-rfkill[5783]: Could not open rfkill device: Could not open RFKILL control device, please verify your installation
Apr 01 08:58:05 fedora30.domain.local systemd[1]: Starting Hostname Service...
Apr 01 08:58:05 fedora30.domain.local gsd-media-keys[5770]: Unable to inhibit keypresses: GDBus.Error:org.freedesktop.DBus.Error.AccessDenied: Permission denied
Apr 01 08:58:05 fedora30.domain.local gsd-xsettings[5789]: Failed to get current display configuration state: GDBus.Error:org.freedesktop.DBus.Error.NameHasNoOwner: Destination does not exist
Apr 01 08:58:05 fedora30.domain.local systemd[1]: Starting Locale Service...
Apr 01 08:58:05 fedora30.domain.local vmware-user.desktop[5856]: vmware-user: could not open /proc/fs/vmblock/dev
Apr 01 08:58:05 fedora30.domain.local gnome-session[5379]: gnome-session-binary[5379]: GLib-CRITICAL: g_child_watch_add_full: assertion 'pid > 0' failed
Apr 01 08:58:05 fedora30.domain.local gnome-session-binary[5379]: GLib-CRITICAL: g_child_watch_add_full: assertion 'pid > 0' failed
Apr 01 08:58:05 fedora30.domain.local systemd[2048]: Created slice dbus\x2d:1.1\x2dorg.freedesktop.problems.applet.slice.
Apr 01 08:58:05 fedora30.domain.local systemd[2048]: Started dbus-:1.1-org.freedesktop.problems.applet@0.service.
Apr 01 08:58:05 fedora30.domain.local gnome-shell[5594]: Some code accessed the property 'CurrentWorkspaceDisplay' on the module 'workspaceDisplays'. That property was defined with 'let' or 'const' inside the module. This was previously supported, but is not correct according to the ES6 standard. Any symbols to be exported from a module must be defined with 'var'. The property access will work as previously for the time being, but please fix your code anyway.
Apr 01 08:58:05 fedora30.domain.local gnome-shell[5594]: Some code accessed the property 'AllWorkspacesDisplay' on the module 'workspaceDisplays'. That property was defined with 'let' or 'const' inside the module. This was previously supported, but is not correct according to the ES6 standard. Any symbols to be exported from a module must be defined with 'var'. The property access will work as previously for the time being, but please fix your code anyway.
Apr 01 08:58:05 fedora30.domain.local gnome-shell[5594]: Some code accessed the property 'IconWorkspaceDisplay' on the module 'workspaceDisplays'. That property was defined with 'let' or 'const' inside the module. This was previously supported, but is not correct according to the ES6 standard. Any symbols to be exported from a module must be defined with 'var'. The property access will work as previously for the time being, but please fix your code anyway.
Apr 01 08:58:05 fedora30.domain.local gnome-shell[5594]: Some code accessed the property 'StyleStore' on the module 'stylesStorage'. That property was defined with 'let' or 'const' inside the module. This was previously supported, but is not correct according to the ES6 standard. Any symbols to be exported from a module must be defined with 'var'. The property access will work as previously for the time being, but please fix your code anyway.
Apr 01 08:58:05 fedora30.domain.local gnome-shell[5594]: loading user theme: /usr/share/themes/Arc-Dark/gnome-shell/gnome-shell.css
Apr 01 08:58:05 fedora30.domain.local cupsd[1189]: REQUEST localhost - - "POST / HTTP/1.1" 200 364 Create-Printer-Subscriptions successful-ok
Apr 01 08:58:05 fedora30.domain.local systemd[1]: Started Hostname Service.
Apr 01 08:58:05 fedora30.domain.local audit[1]: SERVICE_START pid=1 uid=0 auid=4294967295 ses=4294967295 subj=system_u:system_r:init_t:s0 msg='unit=systemd-hostnamed comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
Apr 01 08:58:05 fedora30.domain.local systemd[1]: Started Locale Service.
Apr 01 08:58:05 fedora30.domain.local audit[1]: SERVICE_START pid=1 uid=0 auid=4294967295 ses=4294967295 subj=system_u:system_r:init_t:s0 msg='unit=systemd-localed comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
Apr 01 08:58:05 fedora30.domain.local audit[5685]: AVC avc:  denied  { search } for  pid=5685 comm="geoclue" name="5786" dev="proc" ino=72334 scontext=system_u:system_r:geoclue_t:s0 tcontext=system_u:system_r:unconfined_service_t:s0 tclass=dir permissive=0
Apr 01 08:58:06 fedora30.domain.local gnome-session-binary[5379]: Entering running state
Apr 01 08:58:06 fedora30.domain.local systemd[2048]: Starting Tracker metadata database store and lookup manager...
Apr 01 08:58:06 fedora30.domain.local gsd-color[5786]: failed to obtain org.freedesktop.color-manager.create-profile auth
Apr 01 08:58:06 fedora30.domain.local gsd-color[5786]: failed to obtain org.freedesktop.color-manager.create-profile auth
Apr 01 08:58:06 fedora30.domain.local gsd-color[5786]: failed to obtain org.freedesktop.color-manager.create-profile auth
Apr 01 08:58:06 fedora30.domain.local gsd-color[5786]: failed to obtain org.freedesktop.color-manager.create-profile auth
Apr 01 08:58:06 fedora30.domain.local libcanberra-login-sound.desktop[5889]: Failed to play sound: File or data not found
Apr 01 08:58:06 fedora30.domain.local gsd-media-keys[5770]: g_variant_get_va: assertion 'value != NULL' failed
Apr 01 08:58:06 fedora30.domain.local gsd-media-keys[5770]: g_variant_unref: assertion 'value != NULL' failed
Apr 01 08:58:06 fedora30.domain.local python3[5877]: detected unhandled Python exception in '/usr/libexec/gnome-tweak-tool-lid-inhibitor'
Apr 01 08:58:06 fedora30.domain.local systemd[2048]: Started Tracker metadata database store and lookup manager.
Apr 01 08:58:06 fedora30.domain.local nvidia-settings-user.desktop[5892]: ERROR: Unable to load info from any available system
Apr 01 08:58:06 fedora30.domain.local tracker-store.desktop[5925]: (uint32 1,)
Apr 01 08:58:06 fedora30.domain.local gnome-shell[5594]: Extension apps-menu@gnome-shell-extensions.gcampax.github.com already installed in /home/my_user/.local/share/gnome-shell/extensions/apps-menu@gnome-shell-extensions.gcampax.github.com. /usr/share/gnome-shell/extensions/apps-menu@gnome-shell-extensions.gcampax.github.com will not be loaded
Apr 01 08:58:06 fedora30.domain.local gnome-shell[5594]: Extension launch-new-instance@gnome-shell-extensions.gcampax.github.com already installed in /home/my_user/.local/share/gnome-shell/extensions/launch-new-instance@gnome-shell-extensions.gcampax.github.com. /usr/share/gnome-shell/extensions/launch-new-instance@gnome-shell-extensions.gcampax.github.com will not be loaded
Apr 01 08:58:06 fedora30.domain.local gnome-shell[5594]: Extension places-menu@gnome-shell-extensions.gcampax.github.com already installed in /home/my_user/.local/share/gnome-shell/extensions/places-menu@gnome-shell-extensions.gcampax.github.com. /usr/share/gnome-shell/extensions/places-menu@gnome-shell-extensions.gcampax.github.com will not be loaded
Apr 01 08:58:06 fedora30.domain.local gnome-shell[5594]: Extension user-theme@gnome-shell-extensions.gcampax.github.com already installed in /home/my_user/.local/share/gnome-shell/extensions/user-theme@gnome-shell-extensions.gcampax.github.com. /usr/share/gnome-shell/extensions/user-theme@gnome-shell-extensions.gcampax.github.com will not be loaded
Apr 01 08:58:06 fedora30.domain.local org.gnome.Shell.desktop[5594]: Xlib:  extension "DPMS" missing on display ":10.0".
Apr 01 08:58:06 fedora30.domain.local xrdp-sesman[5000]: rdpRROutputGetProperty:
Apr 01 08:58:06 fedora30.domain.local abrt-server[5979]: Deleting problem directory Python3-2020-04-01-08:58:06-5877 (dup of Python3-2020-01-20-09:07:32-9481)
Apr 01 08:58:06 fedora30.domain.local xrdp-sesman[5000]: rdpRRGetInfo:
Apr 01 08:58:06 fedora30.domain.local systemd[2048]: Starting GLib proxy auto-configuration service...
Apr 01 08:58:06 fedora30.domain.local systemd[2048]: Started GLib proxy auto-configuration service.
Apr 01 08:58:06 fedora30.domain.local systemd[2048]: Starting Evolution calendar service...
Apr 01 08:58:06 fedora30.domain.local gnome-shell[5594]: remove_mnemonics: assertion 'label != NULL' failed
Apr 01 08:58:06 fedora30.domain.local gnome-shell[5594]: length values must specify a unit
Apr 01 08:58:06 fedora30.domain.local gnome-shell[5594]: length values must specify a unit
Apr 01 08:58:06 fedora30.domain.local abrt-notification[6032]: Process 9481 (gnome-tweak-tool-lid-inhibitor) of user 1234567890 encountered an uncaught gi.repository.GLib.GError exception
Apr 01 08:58:06 fedora30.domain.local systemd[2048]: Started Evolution calendar service.
Apr 01 08:58:06 fedora30.domain.local systemd[2048]: Starting Evolution address book service...
Apr 01 08:58:06 fedora30.domain.local systemd[2048]: Started Evolution address book service.
Apr 01 08:58:06 fedora30.domain.local audit[5685]: AVC avc:  denied  { search } for  pid=5685 comm="geoclue" name="5594" dev="proc" ino=75906 scontext=system_u:system_r:geoclue_t:s0 tcontext=system_u:system_r:unconfined_service_t:s0 tclass=dir permissive=0
Apr 01 08:58:06 fedora30.domain.local gnome-shell[5594]: Error looking up permission: GDBus.Error:org.freedesktop.portal.Error.NotFound: No entry for geolocation
Apr 01 08:58:06 fedora30.domain.local gsd-color[5786]: failed to get edid: unable to get EDID for output
Apr 01 08:58:06 fedora30.domain.local gnome-shell[5594]: STACK_OP_ADD: window 0x2000001 already in stack
Apr 01 08:58:06 fedora30.domain.local gnome-shell[5594]: STACK_OP_ADD: window 0x2000001 already in stack
Apr 01 08:58:06 fedora30.domain.local NetworkManager[1173]: <info>  [1585702686.7547] agent-manager: req[0x56414df02400, :1.844/org.gnome.Shell.NetworkAgent/1234567890]: agent registered
Apr 01 08:58:06 fedora30.domain.local gnome-software[5907]: enabled plugins: desktop-categories, fwupd, os-release, packagekit, packagekit-local, packagekit-offline, packagekit-proxy, packagekit-refresh, packagekit-upgrade, packagekit-url-to-app, shell-extensions, appstream, fedora-pkgdb-collections, snap, desktop-menu-path, epiphany, flatpak, hardcoded-blacklist, hardcoded-featured, hardcoded-popular, modalias, packagekit-refine, rewrite-resource, odrs, packagekit-history, provenance, repos, systemd-updates, generic-updates, packagekit-refine-repos, provenance-license, icons, key-colors, key-colors-metadata
Apr 01 08:58:06 fedora30.domain.local gnome-software[5907]: disabled plugins: dummy
Apr 01 08:58:06 fedora30.domain.local gsd-color[5786]: failed to create device: failed to obtain org.freedesktop.color-manager.create-device auth
Apr 01 08:58:06 fedora30.domain.local PackageKit[5739]: uid 1234567890 is trying to obtain org.freedesktop.packagekit.system-sources-refresh auth (only_trusted:0)
Apr 01 08:58:06 fedora30.domain.local PackageKit[5739]: uid 1234567890 obtained auth for org.freedesktop.packagekit.system-sources-refresh
Apr 01 08:58:06 fedora30.domain.local PackageKit[5739]: what-provides transaction /3628_aacbddcc from uid 1234567890 finished with failed after 53ms
Apr 01 08:58:06 fedora30.domain.local xrdp-sesman[5000]: rdpRRGetInfo:
Apr 01 08:58:06 fedora30.domain.local xrdp-sesman[5000]: rdpRRGetInfo:
Apr 01 08:58:07 fedora30.domain.local PackageKit[5739]: resolve transaction /3630_dadcebac from uid 1234567890 finished with failed after 47ms
Apr 01 08:58:07 fedora30.domain.local gnome-shell[5594]: JS ERROR: ReferenceError: Utils is not defined
                                                                    _myCheckWorkspaces@/home/my_user/.local/share/gnome-shell/extensions/putWindow@clemens.lab21.org/moveWorkspace.js:124:9
Apr 01 08:58:07 fedora30.domain.local gnome-shell[5594]: Could not get current seat: No data available
Apr 01 08:58:07 fedora30.domain.local org.gnome.Shell.desktop[5594]: Window manager warning: Overwriting existing binding of keysym 39 with keysym 39 (keycode 12).
Apr 01 08:58:07 fedora30.domain.local org.gnome.Shell.desktop[5594]: Window manager warning: Overwriting existing binding of keysym 34 with keysym 34 (keycode d).
Apr 01 08:58:07 fedora30.domain.local org.gnome.Shell.desktop[5594]: Window manager warning: Overwriting existing binding of keysym 35 with keysym 35 (keycode e).
Apr 01 08:58:07 fedora30.domain.local org.gnome.Shell.desktop[5594]: Window manager warning: Overwriting existing binding of keysym ff53 with keysym ff53 (keycode 66).
Apr 01 08:58:07 fedora30.domain.local org.gnome.Shell.desktop[5594]: Window manager warning: Overwriting existing binding of keysym ff51 with keysym ff51 (keycode 64).
Apr 01 08:58:07 fedora30.domain.local org.gnome.Shell.desktop[5594]: Window manager warning: Overwriting existing binding of keysym 36 with keysym 36 (keycode f).
Apr 01 08:58:07 fedora30.domain.local org.gnome.Shell.desktop[5594]: Window manager warning: Overwriting existing binding of keysym 31 with keysym 31 (keycode a).
Apr 01 08:58:07 fedora30.domain.local org.gnome.Shell.desktop[5594]: Window manager warning: Overwriting existing binding of keysym ff51 with keysym ff51 (keycode 64).
Apr 01 08:58:07 fedora30.domain.local org.gnome.Shell.desktop[5594]: Window manager warning: Overwriting existing binding of keysym 33 with keysym 33 (keycode c).
Apr 01 08:58:07 fedora30.domain.local org.gnome.Shell.desktop[5594]: Window manager warning: Overwriting existing binding of keysym 32 with keysym 32 (keycode b).
Apr 01 08:58:07 fedora30.domain.local org.gnome.Shell.desktop[5594]: Window manager warning: Overwriting existing binding of keysym 38 with keysym 38 (keycode 11).
Apr 01 08:58:07 fedora30.domain.local org.gnome.Shell.desktop[5594]: Window manager warning: Overwriting existing binding of keysym 37 with keysym 37 (keycode 10).
Apr 01 08:58:07 fedora30.domain.local org.gnome.Shell.desktop[5594]: Window manager warning: Overwriting existing binding of keysym ff53 with keysym ff53 (keycode 66).
Apr 01 08:58:08 fedora30.domain.local gnome-shell[5594]: GNOME Shell started at Wed Apr 01 2020 08:58:05 GMT+0800 (AWST)
Apr 01 08:58:08 fedora30.domain.local packagekitd[5739]: LRO_METALINKURL processing failed
Apr 01 08:58:09 fedora30.domain.local packagekitd[5739]: LRO_METALINKURL processing failed
Apr 01 08:58:09 fedora30.domain.local packagekitd[5739]: LRO_METALINKURL processing failed
Apr 01 08:58:09 fedora30.domain.local packagekitd[5739]: LRO_METALINKURL processing failed
Apr 01 08:58:09 fedora30.domain.local xrdp-sesman[5000]: rdpRRGetInfo:
Apr 01 08:58:09 fedora30.domain.local xrdp-sesman[5000]: rdpRRGetInfo:
Apr 01 08:58:10 fedora30.domain.local packagekitd[5739]: LRO_METALINKURL processing failed
Apr 01 08:58:10 fedora30.domain.local packagekitd[5739]: LRO_METALINKURL processing failed
Apr 01 08:58:10 fedora30.domain.local PackageKit[5739]: refresh-cache transaction /3629_edaedbdb from uid 1234567890 finished with failed after 3351ms
Apr 01 08:58:10 fedora30.domain.local gnome-software[5907]: Only 0 apps for recent list, hiding
Apr 01 08:58:10 fedora30.domain.local PackageKit[5739]: what-provides transaction /3631_abbcacea from uid 1234567890 finished with failed after 41ms
Apr 01 08:58:11 fedora30.domain.local PackageKit[5739]: resolve transaction /3632_dcbbdbad from uid 1234567890 finished with failed after 33ms
Apr 01 08:58:11 fedora30.domain.local gnome-software[5907]: hiding category graphics featured applications: found only 4 to show, need at least 9
Apr 01 08:58:12 fedora30.domain.local gnome-software[5907]: g_path_get_basename: assertion 'file_name != NULL' failed
Apr 01 08:58:12 fedora30.domain.local gnome-software[5907]: g_regex_match_full: assertion 'string != NULL' failed
Apr 01 08:58:12 fedora30.domain.local gnome-software[5907]: g_path_get_basename: assertion 'file_name != NULL' failed
Apr 01 08:58:12 fedora30.domain.local gnome-software[5907]: g_regex_match_full: assertion 'string != NULL' failed
Apr 01 08:58:12 fedora30.domain.local PackageKit[5739]: resolve transaction /3633_eaabeecc from uid 1234567890 finished with failed after 34ms
Apr 01 08:58:12 fedora30.domain.local gnome-software[5907]: hiding category games featured applications: found only 0 to show, need at least 9
Apr 01 08:58:12 fedora30.domain.local gnome-software[5907]: g_variant_get_double: assertion 'g_variant_is_of_type (value, G_VARIANT_TYPE_DOUBLE)' failed
Apr 01 08:58:12 fedora30.domain.local PackageKit[5739]: get-updates transaction /3636_dcebdcad from uid 1234567890 finished with failed after 37ms
Apr 01 08:58:13 fedora30.domain.local packagekitd[5739]: LRO_METALINKURL processing failed
Apr 01 08:58:13 fedora30.domain.local PackageKit[5739]: upgrade-system transaction /3637_adbaebab from uid 1234567890 finished with failed after 790ms
Apr 01 08:58:13 fedora30.domain.local gnome-software[5907]: not handling error download-failed for action refine: failed to refine distro upgrade: cannot update repo 'updates-modular': Cannot prepare internal mirrorlist: Curl error (7): Couldn't connect to server for https://mirrors.fedoraproject.org/metalink?repo=updates-released-modular-f31&arch=x86_64 [Failed to connect to mirrors.fedoraproject.org port 443: Connection refused]; Last error: Curl error (7): Couldn't connect to server for https://mirrors.fedoraproject.org/metalink?repo=updates-released-modular-f31&arch=x86_64 [Failed to connect to mirrors.fedoraproject.org port 443: Connection refused]
Apr 01 08:58:13 fedora30.domain.local gnome-software[5907]: Failed to load snap icon: local snap has no icon
Apr 01 08:58:13 fedora30.domain.local PackageKit[5739]: resolve transaction /3638_daadcaab from uid 1234567890 finished with failed after 64ms
```

**Comment by MichaIng** [comment]
Same issue here, found on Debian: https://github.com/MichaIng/DietPi/issues/3022
I'll check and compare the logs to verify that it is the same underlying issue, however symptoms and workaround (RDP over VNC) match.

**Comment by Hoernchen** [comment]
I'm facing the same issue, xorg appers to ignore the inputdevices for some reason, they are not listed in the (sesman)log output for the serverlayout and for some reason default values are then used which break mouse & keyboard - I've tried to mess with the config file, but nothing I do gets xorg to use the damn xrdpkeyb & xrdpmouse devices:

```
[  8285.543] Build Date: 04 September 2020  01:34:27PM
[  8285.543] xorg-server 2:1.20.8-2ubuntu2.4 (For technical support please see http://www.ubuntu.com/support)
[  8285.543] Current version of pixman: 0.38.4
[  8285.543]    Before reporting problems, check http://wiki.x.org
        to make sure that you have the latest version.
[  8285.543] Markers: (--) probed, (**) from config file, (==) default setting,
        (++) from command line, (!!) notice, (II) informational,
        (WW) warning, (EE) error, (NI) not implemented, (??) unknown.
[  8285.543] (++) Log file: ".xorgxrdp.10.log", Time: Fri Oct 16 19:54:56 2020
[  8285.543] (++) Using config file: "/etc/X11/xrdp/xorg.conf"
[  8285.544] (==) Using system config directory "/usr/share/X11/xorg.conf.d"
[  8285.544] (==) ServerLayout "layout"
[  8285.544] (**) |-->Screen "Screen (xrdpdev)" (0)
[  8285.544] (**) |   |-->Monitor "Monitor"
[  8285.544] (**) |   |-->Device "Video Card (xrdpdev)"
[  8285.544] (**) Option "DontVTSwitch" "on"
[  8285.544] (**) Option "AutoAddDevices" "off"
[  8285.544] (**) Not automatically adding devices
[  8285.544] (==) Automatically enabling devices
[  8285.544] (==) Automatically adding GPU devices
[  8285.544] (==) Automatically binding GPU devices
[  8285.544] (==) Max clients allowed: 256, resource mask: 0x1fffff
[  8285.544] (WW) The directory "/usr/share/fonts/X11/cyrillic" does not exist.
[  8285.544]    Entry deleted from font path.
[  8285.544] (WW) The directory "/usr/share/fonts/X11/100dpi/" does not exist.
[  8285.544]    Entry deleted from font path.
[  8285.544] (WW) The directory "/usr/share/fonts/X11/75dpi/" does not exist.
[  8285.544]    Entry deleted from font path.
[  8285.544] (WW) The directory "/usr/share/fonts/X11/100dpi" does not exist.
[  8285.544]    Entry deleted from font path.
[  8285.544] (WW) The directory "/usr/share/fonts/X11/75dpi" does not exist.
[  8285.544]    Entry deleted from font path.
[  8285.544] (==) FontPath set to:
        /usr/share/fonts/X11/misc,
        /usr/share/fonts/X11/Type1,
        built-ins
[  8285.544] (==) ModulePath set to "/usr/lib/xorg/modules"
[  8285.544] (==) |-->Input Device "<default pointer>"
[  8285.544] (==) |-->Input Device "<default keyboard>"
[  8285.544] (==) The core pointer device wasn't specified explicitly in the layout.
        Using the default mouse configuration.
[  8285.544] (==) The core keyboard device wasn't specified explicitly in the layout.
        Using the default keyboard configuration.

 -- snip snip  8< --

[  8285.608] (II) Using input driver 'mouse' for '<default pointer>'
[  8285.608]    Option "CorePointer" "on"
[  8285.608]    Option "driver" "mouse"
[  8285.608]    Option "identifier" "<default pointer>"
[  8285.608] (**) Option "CorePointer" "on"
[  8285.608] (**) <default pointer>: always reports core events
[  8285.608] (WW) <default pointer>: No Device specified, looking for one...
[  8285.608] (EE) <default pointer>: Cannot find which device to use.
[  8285.608] (==) <default pointer>: Protocol: "Auto"
[  8285.608] (**) Option "CorePointer" "on"
[  8285.608] (**) <default pointer>: always reports core events
[  8285.608] (EE) xf86OpenSerial: No Device specified.
[  8285.608] (EE) <default pointer>: cannot open input device
[  8285.608] (EE) PreInit returned 2 for "<default pointer>"
[  8285.608] (II) UnloadModule: "mouse"
[  8285.608] (II) Using input driver 'kbd' for '<default keyboard>'
[  8285.608]    Option "CoreKeyboard" "on"
[  8285.608]    Option "driver" "kbd"
[  8285.608]    Option "identifier" "<default keyboard>"
[  8285.608] (**) Option "CoreKeyboard" "on"
[  8285.608] (**) <default keyboard>: always reports core events
[  8285.608] (**) Option "CoreKeyboard" "on"
[  8285.608] (**) <default keyboard>: always reports core events
[  8285.608] (**) Option "Protocol" "standard"
[  8285.609] (EE) kbd: <default keyboard>: failed to set us as foreground pgrp (Bad file descriptor)
```

**Comment by Hoernchen** [comment]
The fix for me was to add Option "CoreKeyboard"  and Option "CorePointer" to the inputdevices in /etc/X11/xrdp/xorg.conf since the inputdevices in the serverlayout section are apparently ignored so no core pointer and keyboard exists, which leads to forced default devices. No idea why this is suddenly the case, it worked fine on ubuntu 18.04, but broke for me in 20.04.

```
Section "InputDevice"
    Identifier "xrdpKeyboard"
    Driver "xrdpkeyb"
    Option "CoreKeyboard"
EndSection

Section "InputDevice"
    Identifier "xrdpMouse"
    Driver "xrdpmouse"
    Option "CorePointer"
EndSection
```

**Comment by MichaIng** [comment]
Many thanks for sharing this, I'll test it on Debian where this was broken since at least Stretch, I think even Jessie already.

**Comment by Hoernchen** [comment]
Accompnying xorg bug report: https://gitlab.freedesktop.org/xorg/xserver/-/issues/1086

**Comment by esoren** [comment]
I have been troubleshooting an issue where I had no keyboard/mouse using xrdp on a recent Debian install. Adding "CorePointer" and "CoreKeyboard" worked for me. Thanks for sharing -- I would never have found that on my own.

**Comment by MichaIng** [comment]
I have no idea what changed, there was no xrdp or xorgxrdp package upgrade recently, but suddenly on Debian Stretch (with backports packages), Buster and Bullseye keyboard and mouse input works OOTB without the options being present in the config file. Also no xorg package upgrade was done recently, especially not on Debian Stretch 🤔.

@esoren 
Are you on Debian Buster, which desktop do you use and if you are in mood, mind you retesting it without the settings after an `apt update; apt upgrade`? I'm not sure whether I should still add the setting to our implementation, to be failsafe, or can trust in my tests.

**Comment by mudnaes** [comment]
Just confirming that this fixed problem with a newly install Debian 10 installation.  Could login, but no mouse or keyboard working. Though first it was read-only mode, but found this open issue and then got it fixed by adding needed thing to xorg.conf

**Comment by matt335672** [comment]
@MichaIng , @mudnaes , @esoren - this may now be fixed in devel - see #181 and neutrinolabs/xrdp#1784.

I notice in [this post above](164#issuecomment-710357242) by @Hoernchen that:-

```
[  8285.544] (==) ServerLayout "layout"
[  8285.544] (**) |-->Screen "Screen (xrdpdev)" (0)
```

which seems consistent with #181 - the ServerLayout should be `X11 Server`, not `layout`.

Apologies for missing this one first time round.

**Comment by MichaIng** [comment]
Great, thanks for shipping a fix. Yes that all makes sense with the skipped ServerLayout when being defined in override configs. Since the InputDevice were picked up in any case, defining the options there worked as well, but your solution looks cleaner to me. Probably it can be merged as patch into major distributions, like Debian Buster (not sure if Bullseye will get the next version still merged)?

**Comment by matt335672** [comment]
It could be merged, but I think this is unlikely to happen. The Debian/Ubuntu packages are fixed at a particular version, and although they will take security fixes, that's about it. Other launchpad faults I've raised have gone nowhere (see e.g. https://bugs.launchpad.net/ubuntu/+source/xrdp/+bug/1911435). That's a decision they've made on their side, and there's nothing we can do here to affect that. Fedora/CentOS take pretty much anything we care to raise with them.

**Comment by MichaIng** [comment]
Indeed, I experience the same. I think it's something to convince the package maintainer(s) from first to give it more voice inside the Debian community. There are other examples were much larger patches have been merged into stable Debian packages, which were not in particular security related.

The other thing here is that there is not even a related bug report on the Debian bug tracker, I guess since it's mostly an issue that SBC users face, where distros like Raspberry Pi OS, Armbian or us (DietPi) either by default or via config tool allow to disable screen blanking, as such devices are often used in combination with LCDs with a monitoring or kiosk mode app running.

Since Debian Bullseye will be released this summer, is probably better to concentrate on this. I'll raise a request to have it merged there, in case the upstream xorgxrdp release will be too close to Bullseye release.

**Comment by matt335672** [comment]
It's a good point about the Debian bug tracker. We don't normally push things downstream from here unless they're distro-specific, like the one I've linked above, and we've specifically identified them.

The next release of xrdp/xorgxrdp will probably be end of April. Since Bullseye appears to be in soft freeze at the moment, I think it's unlikely they'll be taking that one. If you're happy to raise a request on Debian, that would be very helpful. Feel free to post a link here if you do.

Many thanks

**Comment by MichaIng** [comment]
I opened a merge request at the Debian Gitlab platform. Not sure if it's the most productive way and if the formalities are correct, but I'll give it a week before supporting it with a bug report: https://salsa.debian.org/debian-remote-team/xorgxrdp/-/merge_requests/1

**Comment by metalefty** [comment]
Usually, Debian only accepts security updates but I believe they also accept fatal bug fixes. For example, the daemon process crashes or it doesn't work at all. Personally, I think no keyboard or mouse implies "it doesn't work at all".

**Comment by matt335672** [comment]
Thanks @MichaIng - hopefully you'll have more luck than I've had!

**Comment by MichaIng** [comment]
> I think no keyboard or mouse implies "it doesn't work at all".

That is my hope as well, even that only systems are effected with such an xorg.conf override. Let's see.

**Comment by rbreaves** [comment]
I am having a similar problem.. and as crazy as it may sound I want to have 2 keyboard devices connected to the same computer and use one of them with an xrdp session. I seem to get a lot of Closed socket 19's though..

My wrapper file already has this in it as well/
```
allowed_users=anybody
```

```
May 27 19:06:30 RRZ87MX xrdp[134083]: (134083)(140002481489728)[DEBUG] Closed socket 19 (AF_UNIX)
May 27 19:08:37 RRZ87MX xrdp[134083]: message repeated 36 times: [ (134083)(140002481489728)[DEBUG] Closed socket 19 (AF_UNIX)]
May 27 19:08:40 RRZ87MX xrdp[134083]: (134083)(140002481489728)[DEBUG] Closed socket 19 (AF_UNIX)
May 27 19:08:58 RRZ87MX xrdp[134083]: message repeated 5 times: [ (134083)(140002481489728)[DEBUG] Closed socket 19 (AF_UNIX)]
May 27 19:09:01 RRZ87MX PackageKit: daemon quit
May 27 19:09:01 RRZ87MX systemd[1]: packagekit.service: Succeeded.
May 27 19:09:01 RRZ87MX xrdp[134083]: (134083)(140002481489728)[DEBUG] Closed socket 19 (AF_UNIX)
May 27 19:09:47 RRZ87MX xrdp[134083]: message repeated 13 times: [ (134083)(140002481489728)[DEBUG] Closed socket 19 (AF_UNIX)]
May 27 19:09:50 RRZ87MX xrdp[134083]: (134083)(140002481489728)[DEBUG] xrdp_wm_log_msg: connection problem, giving up
May 27 19:09:50 RRZ87MX xrdp[134083]: (134083)(140002481489728)[DEBUG] Closed socket 19 (AF_UNIX)
May 27 19:09:50 RRZ87MX xrdp[134083]: (134083)(140002481489728)[DEBUG] xrdp_wm_log_msg: some problem
May 27 19:09:50 RRZ87MX xrdp[134083]: (134083)(140002481489728)[DEBUG] xrdp_mm_module_cleanup
May 27 19:09:50 RRZ87MX xrdp[134083]: (134083)(140002481489728)[DEBUG] Closed socket 18 (AF_INET6 ::1 port 42018)
```

**Comment by MichaIng** [comment]
Did you read and try it with the fix provided above? Open your `/etc/X11/xrdp/xorg.conf` and add these lines (or `Option "DefaultServerLayout" "X11 Server"` only to the correct position): https://github.com/neutrinolabs/xorgxrdp/pull/181/files

**Comment by rubyFeedback** [comment]
I found this thread by random googling. I ran into the same problem but independent of xorgxrdp. I think there is something messed up with input-related stuff and evdev or something like that. I don't recall these issues ~2 years ago, but nowadays I can not get my keyboard and mouse to work anymore after compiling xorg-server from source. "startx" works fine, but the mouse and keyboard does not work. Prior to that I could resolve this by compiling all evdev and libinput related parts, but this no longer works either, and I don't get a lot of useful info from xorg-server either. :(

Perhaps it is time to look whether wayland solves these issues ...

**Comment by MichaIng** [comment]
Just a hint for everyone who applied this patch manually and then did or plans to do a distribution upgrade form Debian Buster to Bullseye or similar: Newer `xorgxrdp` requires to load an additional module for glamor support, if the compile was set at least, which is done on Debian Bullseye. Since changing a configuration file manually leads to APT/dpkg not upgrading it without user confirmation, assure that you do the upgrade an re-apply the input fix patch if not shipped by the new distribution package config yet, e.g.:
```
mv /etc/X11/xrdp/xorg.conf.dpkg-dist /etc/X11/xrdp/xorg.conf
grep -q 'DefaultServerLayout' /etc/X11/xrdp/xorg.conf || sed -i '/Section "ServerFlags"/a\    Option "DefaultServerLayout" "X11 Server"' /etc/X11/xrdp/xorg.conf
systemctl restart xrdp
```

**Comment by flurin-conradin** [comment]
I seem to be facing the same issue after updating from Ubuntu 18.04 to Ubuntu 20.04. My mouse and keyboard are not working on the **login screen**. I have tried the fix described [here](https://github.com/neutrinolabs/xorgxrdp/issues/164#issuecomment-710594486) and altered my ` /etc/X11/xrdp/xorg.conf`, without success. 

I am not very savvy to know which different log and configuration files I should look at, however, when I run `sudo systemctl status xrdp` I do get a warning about a keympa file that doesn't match. 

Dez 16 21:28:41 flu xrdp[17665]: (17665)(140012020717376)[INFO ] Loading keymap file /etc/xrdp/km-00000807.ini
Dez 16 21:28:41 flu xrdp[17665]: (17665)(140012020717376)[WARN ] local keymap file for 0x00000807 found and doesn't match built in keymap, using local keymap file

Does anyone have any more pointers?

**Comment by MichaIng** [comment]
Are there any files in `/etc/X11/xorg.conf.d` in your case? Else it may be a different issue.

**Comment by flurin-conradin** [comment]
Thanks for your reply @MichaIng No, this directory does not exist.

**Comment by MichaIng** [comment]
Then I guess your issue is a different one and a newly opened issue would be better to investigate it. This one here was caused by mentioned files overriding the XDRP X11 config partly, and was solved.

**Comment by flurin-conradin** [comment]
thanks @MichaIng. I do highly appreciate your response. I will open another ticket once I get a chance to look into it again so as to gather some more information.

**Comment by erwinlem** [comment]
I have the same problem in freebsd, but is seems to be related to when i have 2 mice connected.
I'm not totally sure but it is something you can investigate.

**Comment by MichaIng** [comment]
Background and workarounds for the original issue are explained throughout this thread. Same as in flurin-conradin's case: If you do not have any override config in `/etc/X11/xorg.conf.d`, your issue is unrelated to this thread and I suggest to open a new one.
